packages/react/src/FronteggProvider.tsx

```diff
@@ -65,6 +66,11 @@ export const Connector: FC<ConnectorProps> = ({ history, appName, ...props }) =>
   }, []);
   ContextHolder.setOnRedirectTo(onRedirectTo);
 
+  //If you want to handle exact routes you can pass it like in example below
+  // const routesShouldKeepQuery = {signUpUrl: "/account/sign-up"}
+  // useQueryKeeper({routes: routesShouldKeepQuery});
+  useQueryKeeper({ routes: app.store.getState().auth.routes });
```

Comment:
This Connector component should not have any rerender mechanisim like useState or useHistory.. and the useQueryKeeper will rerender Connector component for every history change